.changeset/config.json

```diff
@@ -13,7 +13,6 @@
     "@osdk/examples.one.dot.one",
     "@osdk/examples.todoapp",
     "@osdk/tests.verify-fallback-package",
-    "@osdk/shared.test",
```

Comment:
The package "@osdk/foundry-sdk-generator" depends on the ignored package "@osdk/shared.test", but "@osdk/foundry-sdk-generator" is not being ignored. Please add "@osdk/foundry-sdk-generator" to the `ignore` option.

It's a dev dependency :( but still get this error